docs/topics/con_configmap-availability.adoc

```diff
@@ -0,0 +1,168 @@
+[id='using_a_configmap_to_configure_you_spring_boot_application']
+= Using a ConfigMap to configure your {SpringBoot} application
+//ustory: injecting configuration data into your application using a configmap
+
+:deploymentPlatform: Kubernetes
+
+//openshift docs on configmaps
+//https://docs.openshift.org/latest/dev_guide/configmaps.html
+//https://goo.gl/cQTA2B
+//https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-files
+
+link:http://kubernetes.io/docs/user-guide/configmap/[`ConfigMap`] is a {deploymentPlatform} resource type that allows you to externalize the configuration parameters that you pass to your application in the form of key-value pairs or embedded `application.properties` or `application.yaml` templates.
+The `ConfigMapPropertySource` provided by Spring Cloud Kubernetes searches the deployment configuration of your application for a ConfigMap object in which the `metadata.name` property matches the name of your {SpringBoot} application and applies the values to the deployment configuration of your application.
```

Comment:
"Deployment configuration" is also a name of an OpenShift resource, unrelated to this text. Hence it might be confusing. I'd reword to something like:

> The `ConfigMapPropertySource` provided by Spring Cloud Kubernetes looks for a ConfigMap object whose `metadata.name` property matches the name of your {SpringBoot} application and applies the values from the ConfigMap to your application.